config/manager/manager.yaml

```diff
@@ -21,14 +21,18 @@ spec:
         - "--leader-elect"
         - "--feature-gates=EKS=${CAPA_EKS:=true},EKSEnableIAM=${CAPA_EKS_IAM:=false},EKSAllowAddRoles=${CAPA_EKS_ADD_ROLES:=false},EKSFargate=${EXP_EKS_FARGATE:=false},MachinePool=${EXP_MACHINE_POOL:=false},EventBridgeInstanceState=${EVENT_BRIDGE_INSTANCE_STATE:=false},AutoControllerIdentityCreator=${AUTO_CONTROLLER_IDENTITY_CREATOR:=true},BootstrapFormatIgnition=${EXP_BOOTSTRAP_FORMAT_IGNITION:=false},ExternalResourceGC=${EXP_EXTERNAL_RESOURCE_GC:=false},AlternativeGCStrategy=${EXP_ALTERNATIVE_GC_STRATEGY:=false},TagUnmanagedNetworkResources=${TAG_UNMANAGED_NETWORK_RESOURCES:=true},ROSA=${EXP_ROSA:=false}"
         - "--v=${CAPA_LOGLEVEL:=0}"
-        - "--metrics-bind-addr=0.0.0.0:8080"
+        - "--diagnostics-address=${CAPA_DIAGNOSTICS_ADDRESS:=:8443}"
+        - "--insecure-diagnostics=${CAPA_INSECURE_DIAGNOSTICS:=false}"
         image: controller:latest
         imagePullPolicy: Always
         name: manager
         ports:
         - containerPort: 9440
           name: healthz
           protocol: TCP
+        - containerPort: 8443
```

Comment:
Is this a well-known port number defined somewhere?

Reply:
It is: https://github.com/kubernetes-sigs/cluster-api/pull/9264